setup.py

```diff
@@ -37,6 +36,15 @@
     CYTHON_DEPENDENCY,
 ]
 
+if vi < (3, 12):
+    # XXX Revert this change later.
+    #
+    # Python 3.12 is new and there's no aiohttp wheel for it yet.
+    # And pip helfully fails on building a wheel for it and I've
+    # no idea how to disable that.
```

Comment:
There's an environment variable you can set to use the pure Python http parser

Reply:
The problem isn't using aiohttp, it's about not being able to install it with 3.12 in CI

Reply:
I know, the environment variable will let you install it in CI

Reply:
Add this to your workflow and the aiohttp wheel will build and install from pypi

```
      - name: Set AIOHTTP_NO_EXTENSIONS=1 on 3.12
        if: matrix.environment == '3.12'
        run: echo AIOHTTP_NO_EXTENSIONS=1 >> $GITHUB_ENV
        shell: bash
```

Reply:
Hmm, actually, I'll just use `aiohttp== 3.9.0b0`, which was only released 2 days ago.